models/uuv_bluerov2_heavy/uuv_bluerov2_heavy.sdf

```diff
@@ -658,10 +699,10 @@
                 </channel>
             </control_channels>
         </plugin>
-
-        <plugin name="gazebo_imu_plugin" filename="libgazebo_imu_plugin.so">
-            <robotNamespace />
-            <linkName>base_link</linkName>
+        
+        <plugin name='gazebo_imu_plugin' filename='libgazebo_imu_plugin.so'>
+            <robotNamespace/>
+            <linkName>uuv_bluerov2_heavy/imu_link</linkName>
             <imuTopic>/imu</imuTopic>
             <gyroscopeNoiseDensity>0.0003394</gyroscopeNoiseDensity>
             <gyroscopeRandomWalk>3.8785e-05</gyroscopeRandomWalk>
```

Comment:
Can you remove the sdf file in favor of the jinja template?

Reply:
@Jaeyoung-Lim done👍